### ISSUE TITLE
Header Comments

### ISSUE THREAD

**Issue by tapnair** [DVE2000/Dogbone#1]
Hey the comments in the header in this version aren't all there.  Some lines left un commented.

**Comment by caseycrogers** [comment]
I did not comment some functions/classes (the run and stop functions and anything defined inside of them) as those are just specifics to creating a Fusion 360 add-in. Tutorials for these functions (which I largely just copy and pasted from for that part of my code) can be found here under programming interface:
http://fusion360.autodesk.com/resources
However, as of posting this, the links to all the tutorials are broken, but I expect that to be fixed soon.

Let me know if there are any other areas you see as needing comments!

**Comment by tapnair** [comment]
Sorry I just meant some of the lines at the top of your script don't have comment characters on them.  SO it doesn't run for me at first download.

**Comment by caseycrogers** [comment]
Huh that's really weird, which lines? If I click on Dogbone.py and view it from my browser all the lines appear properly commented.

**Comment by tapnair** [comment]
Maybe I just fat fingered it somehow?

-Patrick

On Jun 22, 2015, at 10:24 PM, caseycrogers notifications@github.com wrote:

Huh that's really weird, which lines? If I click on Dogbone.py and view it from my browser all the lines appear properly commented.

**Comment by caseycrogers** [comment]
I did at one point have a version of the code that was missing some comments somewhere in the first few lines, but I quickly pushed a fixed version. Maybe you grabbed it before I updated it.

**Comment by caseycrogers** [comment]
Huh so when I cloned your fork I got the same missing comment #'s that you were getting. I think it's originating from the fact that, for my description, I rather sloppily just edited the .manifest file after the fact to make a multi-line description. It seems like every time someone loads the Add-In into Fusion Fusion copies the description and pastes it to the beginning of the file, forgetting the comment #'s for the second two lines. Not sure how to fix that, definitely a problem though! I'll look into it.
